CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+## How to contribute
+
+Fork the repository and make some changes.
+Once you're done with your changes send a pull request. You have to agree to
+the license. Thanks!
+If you have questions, visit us on irc.freenode.net in #opensuse-factory or
+ask on our mailing list opensuse-factory@opensuse.org
+
+### How to get started
+
+It can be beneficial to learn how openQA job is executed. To setup your own
+openQA instance, please refer to the documentation of [openQA project](https://github.com/os-autoinst/openQA).
+
+Tests developers guide can be found [here](https://github.com/os-autoinst/openQA/blob/master/docs/WritingTests.asciidoc).
+
+If you are looking for a task to start with, check out [openQA Tests](https://progress.opensuse.org/projects/openqatests/issues/)
+Redmine project. Look for tickets with [easy] or [easy-hack] tags.
+
+### Reporting an issue
+
+As mentioned above, we use Redmine as issue tracking tool. In case you found some
+issue with the tests, please do not hesitate to report them [here](https://progress.opensuse.org/projects/openqatests/issues/).
```

Comment:
Better replace the `to report them [here](https://progress.opensuse.org/projects/openqatests/issues/)` with `report [a new issue](https://progress.opensuse.org/projects/openqatests/issues/new)`